src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitSubscriber.java

```diff
@@ -146,10 +148,15 @@ private void handle(String consumeTag, Delivery delivery) {
             for (T value : values) {
                 Objects.requireNonNull(value, "Received value is null");
 
+                String[] labels = extractLabels(value);
+
                 Counter counter = getDeliveryCounter();
-                counter.inc();
                 Counter contentCounter = getContentCounter();
-                contentCounter.inc(extractCountFrom(value));
+
+                if (labels != null && labels.length != 0) {
+                    counter.labels(extractLabels(value)).inc();
+                    contentCounter.labels(extractLabels(value)).inc(extractCountFrom(value));
```

Comment:
Why do we increment the counters only if we have labels? What about the event router that does not have any values?